src/shared/src/components/AltinnDropdown.tsx

```diff
@@ -0,0 +1,73 @@
+import { makeStyles } from '@material-ui/core/styles';
+import cn from 'classnames';
+import * as React from 'react';
+import { AltinnAppTheme } from '../theme';
+import '../../../altinn-app-frontend/src/styles/shared.css';
+
+const useStyles = makeStyles({
+  select: {
+    fontSize: '1.6rem',
+    '&:focus': {
+      outline: `2px solid ${AltinnAppTheme.altinnPalette.primary.blueDark}`,
+    },
+  },
+});
+
+const optionStyle = {
+  display: 'none',
+};
+interface IAltinnDropdownProps {
+  id: string;
+  onChange: (event: React.ChangeEvent<HTMLSelectElement>) => void;
+  onBlur?: (event: React.FocusEvent<HTMLSelectElement>) => void;
+  options: { value: string; label: string }[];
+  value: string;
+  disabled?: boolean;
+  error?: boolean;
+  fullWidth?: boolean;
+  className?: string;
+}
+
+const AltinnDropdown = ({
+  id,
+  onChange,
+  onBlur,
+  options,
+  value,
+  disabled = false,
+  error = false,
+  fullWidth = false,
+  className,
+}: IAltinnDropdownProps) => {
+  const classes = useStyles();
+  return (
+    <React.Fragment>
```

Comment:
The `React.Fragment` element is redundant here, as there as the `select` element is a single element already.